compose/bin/ece-patches

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+if ! bin/cliq ls vendor/bin/ece-patches; then
+    echo "ece-patches tool is not installed, please reinstall."
+fi
+
+bin/cli chmod +x vendor/bin/ece-patches
```

Comment:
We shouldn't have to run chmod each time.
If user has invalid permissions they should run this command as a one off.